contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -351,7 +361,7 @@ func (s *SchedulerServer) serveFrameworkArtifactWithFilename(path string, filena
 }
 
 func (s *SchedulerServer) prepareExecutorInfo(hks hyperkube.Interface) (*mesos.ExecutorInfo, error) {
-	ci := &mesos.CommandInfo{
+		ci := &mesos.CommandInfo{
```

Comment:
Lin, did your `git commit` trigger a go syntax checker?  If not then you need to upgrade your methodology.